### DISCUSSION TITLE
折线图在粒度过多的情况下toptip展示问题

### DISCUSSION THREAD

**Discussion by HenryLee0** [antvis/G2#5848] [Q&A]
### 问题描述

![image](https://github.com/antvis/G2/assets/93959131/0604d631-6967-4bab-93ab-625453055aea)

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by pearmini** [comment]
也许是同一个时间有多条数据？
